site/content/entry/yuzu-progress-report-nov-2020/index.md

```diff
@@ -0,0 +1,169 @@
+
+{{< imgs
+	"./mouse.mp4| Keep in mind that a mouse can't emulate all three dimensional space axis"
+  >}}
+
+As an extra gift for keyboard users, [german77](https://github.com/german77) also added support for [better analog emulation with keyboard inputs.](https://github.com/yuzu-emu/yuzu/pull/4905) With this change, keyboard players can now “drift” the emulated analog stick to get any angle, not just the fixed eight ones you can get by pressing keys combinations.
+
+[german77](https://github.com/german77) also enabled the use of [up to 8 different UDP servers.](https://github.com/yuzu-emu/yuzu/pull/4937) This allows for motion controls for each player.
+
+By [using `NpadStyleSet`,](https://github.com/yuzu-emu/yuzu/pull/4959) [Morph](https://github.com/Morph1984) now limits the available input options depending on the game. For example, in `Pokémon: Let’s Go`, now the only possible options are Handheld, Left Joy-Con or Right Joy-Con.
+
+{{< imgs
+	"./limit.png| No cheating!"
+  >}}
+
+[german77](https://github.com/german77) [tweaked the rumble amplification function,](https://github.com/yuzu-emu/yuzu/pull/4950) making it more linear and less aggressive. This should help when low strength percentage values are used.
+
+By [stubbing both](https://github.com/yuzu-emu/yuzu/pull/5021) `SetNpadCommunicationMode` and `GetNpadCommunicationMode`, [german77](https://github.com/german77) made `Borderlands: Game of the Year Edition` and `Borderlands 2: Game of the Year Edition` playable!
+
+## A silent Guardian, a watchful Protector
+
+[Lioncache](https://github.com/lioncash), our harsh but fair code reviewer. has been [removing](https://github.com/yuzu-emu/yuzu/pull/5028) the `global variable accessor` from the whole project (the linked PR is just the latest section), a work that took months. yuzu used to be able to run a single global system instance, but with this change, yuzu can now create as many emulated instances as needed, all separate from each other. This provides full control over the life cycle of the emulated system, allowing among other things, faster game boot times, and maybe more importantly, forcing the devs to keep up cleaner code, which is easier to maintain in the future.
+
+## Future projects
+
+The `Buffer Cache Rewrite` has been progressing very fast with no delays so far, and Rodrigo is very enthusiastic about starting `Project Hades` after finishing it.
+
+More multicore changes are incoming, along with input improvements...
+
+...And a [certain legend](https://github.com/comex) of iOS hacking has been contributing to the project, time will tell what will this bring us.
```

Comment:
Change `time will tell what will this bring us.` to `time will tell what this will bring us.`

But first and foremost, is it smart to bring attention to their contributions? Have you spoken with them to see if they'd like explicit attention?